src/basic/random-util.c

```diff
@@ -42,11 +42,12 @@
 #include "random-util.h"
 #include "time-util.h"
 
-int dev_urandom(void *p, size_t n) {
+int dev_urandom(void *p, size_t n, bool pseudorandom_ok) {
```

Comment:
so this function now on modern systems will never bother with reading with /dev/urandom, right? (i.e. /dev/urandom is only the fallbackon old kernels?) in that case it really needs a different name, no?